hack/olm/postprocess-bundle-release.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# Copyright (C) 2025 ScyllaDB
+#
+# This script applies post-processing required for a release to an OLM bundle located at the specified target directory.
+
+set -euxEo pipefail
+shopt -s inherit_errexit
+
+source "$( dirname "${BASH_SOURCE[0]}" )/../lib/olm.sh"
```

Comment:
Please consider assigning `dirname "${BASH_SOURCE[0]}` to a local `SCRIPT_DIR` variable, as it's used in two places already.